tests/test_flask.py

```diff
@@ -373,6 +373,21 @@ def test_ical_all(self):
         self.assertEqual(output.data.count('BEGIN:VEVENT'), 15)
         self.assertEqual(output.data.count('END:VEVENT'), 15)
 
+    def test_ical_meeting(self):
+        """ Test the ical_calendar_meeting function. """
+        self.__setup_db()
+
+        output = self.app.get('/ical/calendar/meeting/2/')
+        self.assertEqual(output.status_code, 200)
+        self.assertTrue('BEGIN:VCALENDAR' in output.data)
+        self.assertTrue('SUMMARY:test-meeting2' in output.data)
+        self.assertTrue(
+            'DESCRIPTION:This is another test meeting'
+            in output.data)
+        self.assertTrue('ORGANIZER:pingou' in output.data)
+        self.assertEqual(output.data.count('BEGIN:VEVENT'), 1)
+        self.assertEqual(output.data.count('END:VEVENT'), 1)
```

Comment:
My problem was more about detecting that we are not adding something in the iCal output without realizing it. So by testing the whole iCal output, we're sure there is no suddenly additional data that we didn't plan on.